opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js

```diff
@@ -505,7 +505,7 @@ describe('Elasticsearch pagination', () => {
     let data = await elPaginate(testContext, ADMIN_USER, READ_RELATIONSHIPS_INDICES);
     expect(data).not.toBeNull();
     const groupByIndices = R.groupBy((e) => e.node._index, data.edges);
-    expect(groupByIndices[`${ES_INDEX_PREFIX}_internal_relationships-000001`].length).toEqual(25);
+    expect(groupByIndices[`${ES_INDEX_PREFIX}_internal_relationships-000001`].length).toEqual(33);
```

Comment:
what are the relationships added ? I can count one "has-role" between the new connectors group and connector role, then five "access-to" between the connectors group and the five default markings, which would be +6, what are the last 2 that I didn't count ?

Reply:
There are three new marking-definition in the DATA-TEST-STIX2_v2 file
: 
- TLP:WHITE
- TLP:TEST
- Copyright 2017, The MITRE Corporation

But the TLP:WHITE is resolved by the same id as TLP:CLEAR in identifier.js file

So, we have two new markings ingested and two new relations